.pipelines/cni/pipeline.yaml

```diff
@@ -141,7 +141,7 @@ stages:
       clusterType: swift-byocni-up
       clusterName: linux-cniv2-podsubnet
       nodeCount: ${LINUX_NODE_COUNT}

```

Comment:
We should change these as well ?

Reply:
We can, originally it was setup this way to ensure that all related variables for each cluster were sorted next to each other. With the changes above it is starting to break away for that pattern. Would rather have a singular pattern versus 2. Will change to match.